JDK11 support: tools.jar

heritrix-contrib depends on hbase-client which eventually depends on tools.jar. tools.jar is no longer included in the JDK.

> [ERROR] Failed to execute goal on project heritrix-contrib: Could not resolve dependencies for project org.archive.heritrix:heritrix-contrib:jar:3.4.0-SNAPSHOT: Could not find artifact jdk.tools:jdk.tools:jar:1.6 at specified path /usr/lib/jvm/java-11-openjdk-11.0.3.7-1.fc29.x86_64/../lib/tools.jar -> [Help 1]

--- a/contrib/pom.xml
+++ b/contrib/pom.xml
@@ -26,6 +26,13 @@
 					<artifactId>junit</artifactId>
 					<groupId>junit</groupId>
 				</exclusion>
+				<!-- tools.jar is not available in JDK 11 so exclude it
+				     hbase-client accidentally leaked it as a transitive dependency
+				     https://issues.apache.org/jira/browse/HBASE-13963 -->
+				<exclusion>
+					<groupId>jdk.tools</groupId>
+					<artifactId>jdk.tools</artifactId>
+				</exclusion>
 			</exclusions>
 		</dependency>
 		<dependency>

